<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sent_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Предложения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coll_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сочетания</string>
   </property>
  </widget>
  <widget class="QPushButton" name="same_root_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Однокоренные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="same_root_colls_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сочетания с однокоренными</string>
   </property>
  </widget>
  <widget class="QPushButton" name="same_root_sents_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Предложения с однокоренными</string>
   </property>
  </widget>
  <widget class="QPushButton" name="same_root_conns_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Связи с однокоренным</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conns_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Связи</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
